java/client/src/org/openqa/selenium/chrome/ChromeOptions.java

```diff
@@ -235,6 +236,15 @@ public ChromeOptions setAcceptInsecureCerts(boolean acceptInsecureCerts) {
     return this;
   }
 
+  public ChromeOptions setEnableDownloading(boolean enableDownloading, String downloadPath){
+    if (enableDownloading){
+      setCapability(ENABLE_DOWNLOADING, downloadPath);
+    }else {
+      setCapability(ENABLE_DOWNLOADING, "");
```

Comment:
Setting the capability to `null` will remove it from the map, which is probably more desirable.